src/main/scala/com/advancedtelematic/campaigner/db/Campaigns.scala

```diff
@@ -217,7 +221,7 @@ protected [db] class Campaigns(implicit db: Database, ec: ExecutionContext)
       mainCnt <- deviceUpdateRepo.countByStatus(Set(campaignId)).map(processCounts)
       retryCnt <- deviceUpdateRepo.countByStatus(retryCampaignIds).map(processCounts)
     } yield {
-      val finished = mainCnt.finished  - (retryCnt.rejected + retryCnt.cancelled)
+      val finished = mainCnt.finished - retryCnt.rejected - retryCnt.cancelled
```

Comment:
I know this doesn't change the result, but it made it somehow easier for me to understand where this calculation is coming from.